.github/workflows/build.yml

```diff
@@ -59,7 +59,7 @@ jobs:
           override: true
       - uses: Swatinem/rust-cache@v2
       - name: test
-        run: cargo test --verbose --all-features
+        run: cargo test --workspace --verbose --all-features -- --skip read_table_version_hdfs
```

Comment:
the hdfs test starts up a java environment which takes >1m and makes the whole test suite take much longer, so I'm skipping it for now (could become a nightly test)

Reply:
created #285 to track